assets/js/pages/HostRelevantPatches/HostRelevantPatchesPage.jsx

```diff
@@ -26,29 +26,40 @@ function HostRelevantPatches({ hostName, onNavigate, patches }) {
 
   const [displayedPatches, setDisplayedPatches] = useState(patches);
 
-  const file =
-    window?.URL?.createObjectURL && displayedPatches?.length > 0
-      ? window.URL.createObjectURL(
-          new File(
-            [
-              Papa.unparse(
-                {
-                  fields: [
-                    'advisory_type',
-                    'advisory_name',
-                    'advisory_synopsis',
-                    'update_date',
-                  ],
-                  data: displayedPatches,
-                },
-                { header: true }
-              ),
-            ],
-            `${hostName}-patches.csv`,
-            { type: 'text/csv' }
+  const [csvURL, setCsvURL] = useState(null);
+
+  useEffect(() => {
+    setCsvURL(
+      patches.length > 0
+        ? URL.createObjectURL?.(
```

Comment:
question: Why the `?.` thing? And not `URL.createObjectURL(...)`
General question for all the places where this is happening, which are a few